bio_hansel/quality_check/const.py

```diff
@@ -0,0 +1,12 @@
+FAIL_MESSAGE = "FAIL"
+WARNING_MESSAGE = "WARNING"
+# Errors for Hansel
+MISSING_TILES_ERROR_1A = "Missing Tiles Error 1A"
+MISSING_TILES_ERROR_1B = "Missing Tiles Error 1B"
+MIXED_SAMPLE_ERROR_2A = "Mixed Sample Error 2A"
+INCONSISTENT_RESULTS_ERROR_3A = "Inconsistent Results Error 3A"
+INCONSISTENT_RESULTS_ERROR_3B = "Inconsistent Results Error 3B"
+INTERMEDIATE_SUBTYPE_WARNING = "Intermediate Subtype Warning"
+# Thresholds for tile checking.
+MAX_TILES_THRESHOLD = 0.01
```

Comment:
I think it'd be a good idea to make these thresholds and other currently hardcoded values command-line opts with these values as defaults. We can then have the values passed around in some object/dict.

e.g.
```python
parser.add_argument('--max-tiles-threshold',
                        type=float,
                        default=0.01,
                        help='Max tiles threshold percent (0.01 -> 1%)')
```

Reply:
Agreed, I'll change the hardcoded values to command line arguments.

Reply:
Do you think creating a class would be a good idea in case there are more wanted arguments? Or should I just use a dict to pass the values to the QC module?

Reply:
I think an `attrs` class would be a good idea (class name idea: `SubtypingParams`?). You could bundle a bunch of params for various thresholds into the class like the min/max tiles thresholds as well as the QC specific thresholds. The defaults could be whatever the hardcoded or default cmd line opt values are. The object could be passed to the `subtype_*` functions and the QC functions rather than adding a new param for each new threshold that we could up with.